<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1037</width>
    <height>726</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>1021</width>
      <height>651</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Shape::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Shadow::Raised</enum>
    </property>
    <widget class="QPushButton" name="disable_avp_button">
     <property name="geometry">
      <rect>
       <x>730</x>
       <y>40</y>
       <width>75</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Disable AVP</string>
     </property>
    </widget>
    <widget class="QPushButton" name="enable_avp_button">
     <property name="geometry">
      <rect>
       <x>650</x>
       <y>40</y>
       <width>75</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Enable AVP</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="avp_status">
     <property name="geometry">
      <rect>
       <x>810</x>
       <y>40</y>
       <width>151</width>
       <height>30</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>30</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="set_frequency_button">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>150</y>
       <width>211</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Set Frequency</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="set_frequency">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>150</y>
       <width>231</width>
       <height>30</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>500</width>
       <height>30</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="get_name_button">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>320</y>
       <width>351</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Get Device Name</string>
     </property>
    </widget>
    <widget class="QPushButton" name="get_info_button">
     <property name="geometry">
      <rect>
       <x>600</x>
       <y>320</y>
       <width>351</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Get Full Info</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="response_text">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>400</y>
       <width>1001</width>
       <height>241</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="set_level_button">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>190</y>
       <width>211</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Set Level</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="set_level">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>190</y>
       <width>231</width>
       <height>30</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>500</width>
       <height>30</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="set_bias_button">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>230</y>
       <width>211</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Set Bias</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="set_bias">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>230</y>
       <width>231</width>
       <height>30</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>500</width>
       <height>30</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="connection_button">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>40</y>
       <width>75</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Connect</string>
     </property>
    </widget>
    <widget class="QComboBox" name="combo_com_port">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>40</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>COM1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>COM2</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>COM3</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>COM4</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>COM5</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>COM6</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>COM7</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>COM8</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>COM9</string>
      </property>
     </item>
    </widget>
    <widget class="QPushButton" name="mode_button">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>80</y>
       <width>75</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Set</string>
     </property>
    </widget>
    <widget class="QComboBox" name="combo_mode">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>80</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>R-X</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>C-f</string>
      </property>
     </item>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1037</width>
     <height>33</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
